_includes/tutorials/generate-test-data-streams/kafka/markup/dev/create-connector-02.adoc

```diff
@@ -1,46 +1,34 @@
-Now let's see how you can generate streams of data that is related and consistent. 
+Now let's see how you can generate streams of data that is related and consistent. You are going to use https://github.com/confluentinc/kafka-connect-datagen/tree/master#use-a-bundled-schema-specification[bundled schemas] available in the Kafka Connect `DatagenConnector`.
 
-Consider an example where we want to generate some dummy data about network devices. The stream of events would have fields such as
 
--> The MAC address of the device
+Consider an example where we want to generate some dummy data for tracking sensors on a fleet of delivery trucks.  There will be two streams of data,
```

Comment:
missing second stream description in the second sentence?  e.g.:

There will be two streams of data: (1) the current location of each truck, and (2) truck sensor information from each truck.